design-system-docs/src/_component_docs/summary-list.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Summary list
+---
+
```

Comment:
This is a simple component but is there a paragraph we can include at the start to say where it can be used? I'm aware not of our components have this (they probably should 😬) but I wonder if the design team can help with a definition of the use cases for the component (e.g to be used to summarise form data, etc). It can be a part of a separate PR though.

Reply:
yep, I've added this now as turns out our designer had prepared something :)